patches/node/build_add_gn_build_files.patch

```diff
@@ -7,10 +7,10 @@ This adds GN build files for Node, so we don't have to build with GYP.
 
 diff --git a/BUILD.gn b/BUILD.gn
 new file mode 100644
-index 0000000000000000000000000000000000000000..9e34a074cfa7dec61c4e11821ba5f1969f393dfb
+index 0000000000000000000000000000000000000000..747df4dc18ed11edada38056f2ce7c9300000000
```

Comment:
```suggestion
index 0000000000000000000000000000000000000000..3ae9b93649bb404fbf04dea52ecaa7d664ff03ee
```
hopefully, this will fix the circleci failure

Reply:
Are these actual hashes, or just an arbitrary unique ID?

Reply:
these are actual commit shas